### ISSUE TITLE
Code doesn't compile

### ISSUE THREAD

**Issue by tbisanz** [eutelescope/eutelescope#424]
@jha92 reported in #398 that compilation is broken - follow up on this!

**Comment by jha92** [comment]
When I ran with my older setup and tried to compile the newest master version, I got errors which were previously marked as warnings.
I started now from a fresh installation and could not reproduce anything worrisome, so everything fine.
Just to check, I switched back to my old version (it is "Latest commit c9d54be on Mar 13" i.e. pull request #376 ), both compilation logs can be found here:
[compilation_master_old.log](https://github.com/eutelescope/eutelescope/files/2077352/compilation_master_old.log)
[compilation_master.log](https://github.com/eutelescope/eutelescope/files/2077353/compilation_master.log)
So, in conclusion: not sure what happened, but know it is okay. Suggest to close this here.
